fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTabLayouts.kt

```diff
@@ -49,6 +62,8 @@ import kotlin.math.max
  * @param onTabMediaClick Invoked when the user interacts with a tab's media controls.
  * @param onTabClick Invoked when the user clicks on a tab.
  * @param onTabLongClick Invoked when the user long clicks a tab.
+ * @param onMove Invoked when the user moves a tab.
+ * @param onExitSelectionMode Invoked when starting to drag a tab.
```

Comment:
`onExitSelectionMode` makes sense for the lambda in the nodes above this one but maybe not here based on the kdoc and the trigger for it.

```suggestion
 * @param onTabDragStart Invoked when starting to drag a tab.
```